packages/suite/src/utils/suite/tor.ts

```diff
@@ -46,3 +47,33 @@ export const torFetch = (input: RequestInfo, init?: RequestInit | undefined) =>
 
     return baseFetch(input, init);
 };
+
+export const getIsTorEnabled = (torStatus: TorStatus) => {
```

Comment:
Note: By using enum logic in Redux state, we will lose "human-readable" redux store. Opinions @marekrjpolak 
 @matejkriz ? I think that this will be the first occurrence.

Reply:
I can just use a string enum :)

Reply:
done a724af709a93334f0ab39778ed7f55e13306db96